app/controllers/boards_controller.rb

```diff
@@ -1,10 +1,19 @@
 class BoardsController < ApplicationController
-  skip_before_action :authenticate_user!, only: [:show]
-  before_action :set_board, except: [:index, :new, :create]
-  before_action :require_same_user, only: [:destroy]
+  authorize_persona class_name: "User"
+  grant(
+    user: [:show, :index],
+    manager: :all,
+    admin: :all,
+  )
+  before_action :set_author_from_url, only: [:index, :create, :new]
+  before_action :set_board_from_url, only: [:show, :edit, :update, :destroy]
+  before_action :set_author_from_board, only: [:edit, :update, :destroy, :show]
+  
+  before_action :same_user_as_author, only: [:create, :new, :edit, :update, :destroy]
```

Comment:
same here.

Reply:
This code was also updated to use excepts instead of "only".